src/test/test-reread-partition-table.c

```diff
@@ -47,7 +47,7 @@ TEST(rereadpt) {
 
         if (detect_container() > 0)
                 return (void) log_tests_skipped("test not available in container");
-        if (geteuid() != 0 || have_effective_cap(CAP_SYS_ADMIN) <= 0)
+        if (have_effective_cap(CAP_SYS_ADMIN) <= 0)
```

Comment:
have you double checked that running_in_chroot still works correctly in a chroot?